src/main/java/me/jellysquid/mods/sodium/client/gl/shader/GlProgram.java

```diff
@@ -87,7 +88,7 @@ public Builder attachShader(GlShader shader) {
                 LOGGER.warn("Program link log for " + this.name + ": " + log);
             }
 
-            int result = GL20C.glGetProgrami(this.program, GL20C.GL_LINK_STATUS);
```

Comment:
lmao @jellysquid3 why are you calling gl 2.0

Reply:
tehc, stop shitposting

Reply:
sorry